src/datatype.jl

```diff
@@ -0,0 +1,36 @@
+# custom data tyoe to hold ImageHDU
+struct CCDData{T,M<:AbstractMatrix{T}} <: AbstractMatrix{T}
+    data::M
+    hdr::FITSHeader
+end
```

Comment:
let's do
```julia
abstract type AbstractCCDData{T} <: AbstractMatrix{T}
struct CCDData{T,M<:AbstractMatrix{T}} <: AbstractCCDData{T}

end
```

This will allow future users to be able to easily extend and make their own `CCDData` type!